docs/decisions/0015-i18n-support.rst

```diff
@@ -0,0 +1,46 @@
+15. Internationalization support.
+---------------------------------
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+Paragon's components are extensively used in applications that support i18n.
+Currently consumers of Paragon need to handle i18n themselves, i.e. pass translated
+strings to components as props. Furthermore, some of the components have hardcoded strings in them which do not
```

Comment:
Might change to be (e.g., pass translated strings to components as props, or pass components like `FormattedMessage` from react-intl as nodes).

Reply:
This sound better, updated, thanks!